source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/include/ACOActionOverflow.h

```diff
@@ -1 +1 @@
-../ACOActionOverflow.h
\ No newline at end of file
+../ ACOActionOverflow.h
```

Comment:
is the space between the slash "/" and the file name correct?

Reply:
I don't know about it. Can you do a testing on your local machine? I didn't see errors in build pipeline, and I guess it's supported.

Reply:
yes, this is a bug. I tested it. the symbolic link was broken.

Reply:
Thanks. I revert the change on include folder. Do you know why the pipeline doesn't catch the problem?

Reply:
these files are used by swift package manager. and not directly by ci pipeline. it's a good idea, i just added cocoapod spec check, we can add this spm check in next eim.